actions/lib/formatters.py

```diff
@@ -14,6 +15,45 @@
 ]
 
 
+def branch_protection_to_dict(branch_protection):
+    result = {}
+    for attr in branch_protection_attributes:
+
+        if hasattr(branch_protection, attr):
```

Comment:
Could avoid one level of branching here by doing:
```suggestion
        if not hasattr(branch_protection, attr):
            continue
```